span-normalizer/src/test/java/org/hypertrace/core/spannormalizer/JaegerSpanNormalizerTest.java

```diff
@@ -116,4 +121,21 @@ public void invalidConfigCombination1() {
                   "Both processor.tenantIdTagKey and processor.defaultTenantId configs shouldn't exist at same time"));
     }
   }
+
+  @Test
+  public void testServiceNameAddededToEvent() {
+    Map<String, Object> configs = new HashMap<>(getCommonConfig());
+    configs.putAll(
+        Map.of(
+            "processor",
+            Map.of(
+                "defaultTenantId", "tenant-1")));
+    JaegerSpanNormalizer normalizer = JaegerSpanNormalizer.get(ConfigFactory.parseMap(configs));
+    Process process = Process.newBuilder().setServiceName("testService").build();
```

Comment:
Can you add a test for when we don't set `serviceName` in the process?
So, will able to set assertNull for rawSpan.getEvent().getServiceName().